tools/rosbag/src/rosbag/bag.py

```diff
@@ -1475,6 +1475,7 @@ def __str__(self):
             return 'compression: %s, size: %d, uncompressed: %d' % (self.compression, self.compressed_size, self.uncompressed_size)
 
 class ComparableMixin(object):
+    __slots__ = []
```

Comment:
Can you please add a newline after all of the added `__slots__` to follow PEP8 (even if the existing file is not following it everywhere but at least new stuff should do so). Thanks.